types/types.go

```diff
@@ -130,6 +134,9 @@ const (
 
 	DeleteCustomResourceOnly = "delete-custom-resource-only"
 
+	DeleteBackupTargetFromLonghorn = "delete-backup-target-from-longhorn"
+	UpdateBackupTargetFromLonghorn = "update-backup-target-from-longhorn"
```

Comment:
What's the purposes of the two labels? Leave a comment here.

Reply:
Removed unused label, and added the comment for purposes. Thanks